integration-test-groups/foundation/language/pom.xml

```diff
@@ -31,6 +31,10 @@
     <description>Integration tests for Camel Quarkus Language extension</description>
 
     <dependencies>
+		<dependency>
```

Comment:
Isn't this caught by mvn ...-Pformat ?

Reply:
I just ran the format command again and it didn't pick it up. I'll fix that manually.